src/content/contributing/building_testing/_index.en.md

```diff
@@ -45,29 +45,68 @@ To run Go unit tests:
 make test
 ```
 
-### End-to-end tests
+### Multi-Cluster KIND Based Environment {#clusters}
 
-To run functional tests with a full multicluster deployment:
+Shipyard provides a basic target that creates a KIND based multi-cluster
+environment, without any special deployment (apart from the default K8s):
+
+```
+make clusters
+```
+
+Optionally, you can specify flags to control the clusters being deployed:
+* k8s_version - Controls which K8s version the KIND nodes deploy with
+* globalnet - Deploys clusters that have overlapping CIDRs, to be used with
```

Comment:
are we also adding globalnet size and globalnet cidr (soon to be available) options too?

Reply:
They will go as part of extra deploy flags, that section has not been written yet